build.sbt

```diff
@@ -1438,6 +1438,5 @@ ThisBuild / dependencyOverrides ++= Seq(
   "io.opencensus" % "opencensus-contrib-grpc-metrics" % opencensusVersion,
   "org.checkerframework" % "checker-qual" % checkerFrameworkVersion,
   "org.codehaus.mojo" % "animal-sniffer-annotations" % animalSnifferAnnotationsVersion,
-  "org.scala-lang.modules" %% "scala-xml" % scalaXmlVersion,
-  "org.threeten" % "threetenbp" % threetenbpVersion
```

Comment:
We're indeed not supporting java 6/7 since long time. This dep was added because it showed as conflict in the `evicted` in previous beam version, so we favored the one shipped with beam.
Since 2.41, this dep is not used anymore by beam, and there is no conflicts. Hence removing the definition here